docs/io-handbook/how-to-deploy.md

```diff
@@ -0,0 +1,88 @@
+---
+title: How to deploy
```

Comment:
The guide actually defines deploy procedure for runnable applications. Do we want to differentiate for different types of software? We have
* npm packages
* the App itself

Reply:
I'd rather write other specific guides for npm packages and the App 👍

Reply:
I agree.